app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -42,7 +42,8 @@ object FeatureFlags {
     /**
      * Enables the new MediaSession API.
      */
-    val newMediaSessionApi = Config.channel.isNightlyOrDebug
+    @Suppress("MayBeConst")
+    val newMediaSessionApi = true
```

Comment:
Any reason we shouldn't just do `const val` for the time being even if we do end up turning it back into `isNightlyOrDebug`

Reply:
Sorry, I should have been more clear.  This is so we can turn it off quickly if any issue happens in beta.  I have another issue to track removing this completely https://github.com/mozilla-mobile/fenix/issues/17174.  Thanks,